migrations/014.do.top-measurement-participants-mv.sql

```diff
@@ -0,0 +1,36 @@
+CREATE MATERIALIZED VIEW top_measurement_participants_yesterday_view AS
```

Comment:
Was hard to pick a good name, since this exact thing already exists (in terms of the data stored, not how it retrieves it). Maybe you have a better approach in mind?

Reply:
The difference in retrieval is basically that we get the data from two tables now, `recent_station_details` and `recent_participant_subnets`, instead of just one before (`daily_stations`)

I can't think of a good name related to that though. Especially since this will last long-term, while the other will not, so any specifics about the differences will be lost to time (in one year, "Why did we name this "top_measurement_participants_yesterday_multiple_tables?" haha)

I like the `_deprecated` idea for the old one, but it would require changing the name for the existing table in this migration script though, and then everywhere else that uses it (including creating a new PR to change `spark-stats`)

Reply:
How about doing the simplest thing and appending `2` or `_v2` to the table name? It's not the prettiest thing to do, but it gives us a clear path forward if we ever need v3 or v4 of this.

Reply:
This sounds good to me. @juliangruber ?

Reply:
See https://github.com/filecoin-station/spark-evaluate/pull/341#discussion_r1738941823 - I think we don't need the second materialised view and can simply redefine the existing one.